### PR TITLE
Fix Stuff

### PR DESCRIPTION
**Pull request by hapuc777**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/06-chapter-5.adoc [hunk 1]
```diff
@@ -752,18 +752,7 @@ had recently undergone a change.
...
-and now I was enabled through holy help to cast off a great burden.
-Blessed, praised and magnified be the name of Israel`'s Shepherd.
-He brings low and raises up, and is worthy of all glory and honor forever.
-I was largely opened in this meeting to declare against infidelity, atheism,
-and all unrighteousness; and had to bear my testimony against those works of darkness,
-called spiritual rappings, and the workers thereof; stating my belief,
-that it was the devil and his agents who carried it on, and were the originators thereof.
-A young man, a stranger to us,
-expressed his satisfaction with what he had heard delivered.
+and parted under solemn feelings.
```
Comment (hapuc777) [inline journal/modernized/06-chapter-5.adoc:755]: This big chunk that I erased didn't belong there (bottom of page 111). Belongs to another paragraph on top of the p. 113 (and it's actually already there!)

File: journal/modernized/09-chapter-8.adoc [hunk 2]
```diff
@@ -995,8 +995,7 @@ because it was not possible he should be holden of them:
 He is Lord of lords and King of kings;
 He can and will do great and wonderful things for those who put their trust in Him.
 Blessed, praised and magnified be his adorable name, says my soul,
-forever and forever.ling to receive counsel from the very least child,
-willing to take advice as well as give.
+forever and forever.
```
Comment (hapuc777) [inline journal/modernized/09-chapter-8.adoc:998]: same here.... what I deleted (top of p. 219) was actually missing on the next page (220), on second paragraph. I already added it and checked with the original book from google.

File: journal/modernized/13-chapter-12.adoc [hunk 3]
```diff
@@ -78,9 +78,6 @@ as this is obeyed and followed we shall experience our spirits refreshed and
 strengthened by the life-giving presence of Him who is the minister of the
 sanctuary and true tabernacle which God has pitched,
 and not man.
-you not be pleased to succor me, and help me;
-blot out my transgressions from your book of remembrance for Jesus Christ`'s sake.
-Amen, and amen.
```
Comment (hapuc777) [inline journal/modernized/13-chapter-12.adoc:83]: again here... the chunck deleted (from p. 323) is already on the third paragraph (same page). Also checked with the original book from google.

**Comment by Henderjay** [comment]
Hey Keren. Those corrections were great! I merged them into the master. Thanks for your help with this!!!!

**Comment by hapuc777** [comment]
Great, glad to help! :)

**Pull request merged**